homeassistant/components/accuweather/strings.json

```diff
@@ -22,6 +22,15 @@
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
     }
   },
+  "entity": {
```

Comment:
Should this be name spaced by entity domain?

Reply:
I don't think we can do that for the top-key. If we do that, it should be included in the lookup for `<domain>__<translation_key>`

Reply:
Could also be a subkey under entity: entity -> domain -> key.

It would potentially allow for having different json schema validations depending on the domain of the entity in the future. Additionally it would help/preventing collisions in key across multiple domains.

Reply:
Can you explain @frenck when there are likely to be collisions under `integration` -> `entity` -> `translation_key`? Do you mean an integration may want to use the same translation key for entities from multiple domains, which is possible for example with `unique_id` and `entity_id`, so it's unexpected to restrict it here?

Reply:
My suggestion is to make it something like this:

```json
  "entity": {
    "sensor":
      "pressure_tendency": {
        "state": {
          "steady": "Steady",
          "rising": "Rising",
          "falling": "Falling"
        }
      }
    }
  },
```

This makes the translation key unique per entity domain, making them unique within each domain (just like we do with `unique_id`).

It might help with adding JSON schema validation to our CI/translations files that can take the domain into account (in the possible future).

Reply:
Right, this makes sense 👍

Reply:
Should we even be doing this in `strings.json` instead of `strings.sensor.json`?

Reply:
I personally do not have a strong opinion on that. In general, for most existing integrations `string.<domain>.json` are pretty small. Not sure if it adds a real benefit.

Reply:
I think either `strings.json` or `strings.sensor.json` work, but two questions:
Does referencing a key from `strings.sensor.json` from (for example) `strings.binary_sensor.json` work though?
Does it come natural that the climate base component's translations would then be in `climate/strings.climate.json`?

Reply:
Thought baotu it more. `strings.json` should be used as this is translations provided by the integration for the integration. `strings.sensor.json` is to provide translations for another integration because we were abusing device classes.

Reply:
OK 👍